### PR TITLE
Disable trying to use dump-bundle with integrity-block

### PR DESCRIPTION
**Pull request by sonkkeli**
peletskyi@ 's opinion was that this approach would be better than trusting on the web bundle length vs file size. Let me know if you disagree :)

### CODE REVIEW

File: go/bundle/cmd/dump-bundle/main.go [hunk 1]
```diff
@@ -91,7 +93,30 @@ func isTextType(mimeType string) bool {
 	return strings.HasPrefix(m, "text/") || m == "application/javascript"
 }
 
+func checkDoesNotHaveIntegrityBlock(path string) error {
```
Comment (irori) [inline go/bundle/cmd/dump-bundle/main.go:96]: Can you put this check into `ReadBundleFromFile()` so that the bundle file doesn't have to be opened twice?

**Review by irori — COMMENTED** [review]
The approach lgtm.

**Comment by sonkkeli** [comment]
PTAL :)

**Review by irori — APPROVED** [review]
lgtm

**Pull request merged**